.github/workflows/dependent-repos.yml

```diff
@@ -2,7 +2,9 @@ name: Check that repositories submoduling us will still build after this PR
 
 on:
   pull_request:
-    branches: [ "master" ]
+    branches:
+      - master
+      - main
```

Comment:
`ci.ymll` already has both branches listed, pretty sure this will allow us to flip the switch on renaming and it will just work.